docs/dietpi_tools.md

```diff
@@ -329,13 +329,16 @@ It is one of the core tools, enabling you to install or uninstall one or more [*
 
     There are a couple of general commands which have a shortcut in DietPi:
 
-    - `cpu` - Print CPU clocks, temperature and governor info
+    - `cpu` - Print CPU clocks, temperature and governor info. Sample output:
+
+        ![DietPi-Software CPU info screenshot](assets/images/dietpi-tools-cpuinfo.png)
```

Comment:
Instead of a screenshot, we could also add a code box and copy&paste the console output, actually 🤔. Less images => faster page loading. However, colours are not easily possible then, not sure if there is any way to manually colour code content via Markdown or one of the extensions.

Reply:
I have not found a text color option, especially not in a code block.  
So let's keep it 'nicer' for the readers and not 100% speed optimized.